### PR TITLE
feat(recipient_app): Updated login registration flow

### PR DESCRIPTION
**Pull request by MDikkii**

### CODE REVIEW

**Review by ssandino — APPROVED** [review]

File: recipients_app/lib/data/repositories/user_repository.dart [hunk 1]
```diff
@@ -59,18 +59,21 @@ class UserRepository {
...
     required String phoneNumber,
-    required Function(String) onCodeSend,
+    required Function(String, int?) onCodeSend,
     required Function(FirebaseAuthException) onVerificationFailed,
     required Function(PhoneAuthCredential) onVerificationCompleted,
+    required int? forceResendingToken,
   }) async {
     await firebaseAuth.verifyPhoneNumber(
       phoneNumber: phoneNumber,
+      forceResendingToken: forceResendingToken,
```
Comment (MDikkii) [inline recipients_app/lib/data/repositories/user_repository.dart:69]: Added this to make sure that the same phone can be provided twice even if `auto retrieval timeout` hasn't passed yet.

File: recipients_app/lib/ui/inputs/input_text.dart [hunk 2]
```diff
@@ -37,6 +39,7 @@ class InputText extends StatelessWidget {
             // labelText: hintText,
             suffixIcon: suffixIcon,
             floatingLabelBehavior: FloatingLabelBehavior.never,
+            counter: const SizedBox.shrink()
```
Comment (novas1r1) [inline recipients_app/lib/ui/inputs/input_text.dart:42]: What is this good for and why is it empty?

**Review by novas1r1 — APPROVED** [review]

File: recipients_app/lib/ui/inputs/input_text.dart (hunk 2 above)
Comment (MDikkii) [reply recipients_app/lib/ui/inputs/input_text.dart:42]: When the characters limit is added Flutter is adding the counter on the bottom of the text input automatically. Which is not what we want AFAIK. One of the solutions (which also make it not adding extra space to the bottom of input fields is like above.

Comment (MDikkii) [reply recipients_app/lib/ui/inputs/input_text.dart:42]: Source https://stackoverflow.com/a/58819500

Comment (novas1r1) [reply recipients_app/lib/ui/inputs/input_text.dart:42]: maybe add this as a comment, otherwise we will stumble over this again :D?

Comment (MDikkii) [reply recipients_app/lib/ui/inputs/input_text.dart:42]: sure 👍

**Pull request merged**